src/node_options.cc

```diff
@@ -640,6 +640,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "set environment variables from supplied file",
             &EnvironmentOptions::env_file);
   Implies("--env-file", "[has_env_file_string]");
+  AddOption("--env-file-if-exists",
+            "set environment variables from supplied file",
```

Comment:
```suggestion
            "set environment variables from supplied file if it exists",
```

Reply:
I think that may be obvious enough from the flag itself? I can add it if you don't agree

Reply:
It’s a nit, so you can certainly ignore it. IMO, when looking at the `node --help` output, it’s helpful to have a uniqueness – arguably the help test is not necessary at all, folks can probably infer what the flag does without looking at the help text, with or without my suggestion.